docs/src/conf.py

```diff
@@ -95,9 +95,12 @@ def autolog(message):
 
 # Create a variable that can be inserted in the rst "|copyright_years|".
 # You can add more variables here if needed.
+
+build_python_version = ".".join([str(i) for i in sys.version_info[:3]])
+
 rst_epilog = f"""
 .. |copyright_years| replace:: 2010 - {upper_copy_year}
```

Comment:
@tkknight Could we take the opportunity to rationalise this a wee bit, now that we're in this space...

i.e., Could you change **line 72** from this:
```python
upper_copy_year = datetime.datetime.now().year
```
to this:
```python
copyright_years = f"2010 - {datetime.datetime.now().year}"
```
Then simply use `copyright_years` on **line 102** and **line 231** e.g.,
```python
rst_epilog = f"""
.. |copyright_years| replace:: {copyright_years}
```
and
```python
html_context = {
    "copyright_years": copyright_years,
```

Reply:
Done.  See 793a219.

I also added the **copyright_years** to the **copyright** variable to it is now included in the footer of every page.